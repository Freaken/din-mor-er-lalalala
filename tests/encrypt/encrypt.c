#include <stdio.h>

// Dette c-eksempel beregner det samme som encrypt.jan

int main() {
	int x,y;
	int key;

	scanf("%d", &key);
	scanf("%d", &x);

	for(y = 0; y < 19; y++) {
		if(x % 2 == 0) {
			x/=2;
		} else {
			x/=2;
			x+=key;
		}
	}
		

	printf("%d\n", key);
	printf("%d\n", x);
}
