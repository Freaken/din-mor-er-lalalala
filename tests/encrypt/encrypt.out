555555555
937644535
